dashboard/bin/sqs/process_queues

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+# A service for processing a set of SQS queues defined by a config file.
+# Usage ./process_queues QUEUE_CONFIG.JSON
+
+require_relative '../../config/environment'
+
+require 'json'
+require 'optparse'
+require 'ostruct'
+require 'pp'
```

Comment:
are `optparse`, `ostruct` and `pp` referenced here for upcoming or since-deleted functionality?